maven-release-plugin/src/main/java/org/apache/maven/plugins/release/AbstractScmReleaseMojo.java

```diff
@@ -89,6 +89,12 @@
     @Parameter( defaultValue = "[maven-release-plugin] ", property = "scmCommentPrefix" )
     private String scmCommentPrefix;
 
+    /**
+     * When cloning a repository if it should be a shallow clone or a full clone.
+     */
+    @Parameter( defaultValue = "true", property = "shallowClone" )
+    private boolean shallowClone = true;
```

Comment:
Shouldn't this be `scmShallowClone` for consistency reasons?

Reply:
yes you are right